keps/sig-apps/1847-autoremove-statefulset-pvcs/README.md

```diff
@@ -394,10 +412,17 @@ This features adds a new field to the StatefulSet. The default value for the new
 maintains the existing behavior of StatefulSets.
 
 On a downgrade, the `PersistentVolumeClaimRetentionPolicy` field will be hidden on
-any StatefulSets. The behavior in this case will be identical to mutating they
+any StatefulSets. The behavior in this case will be identical to mutating the
 policy field to `Retain`, as described above, including the edge cases
 introduced if this is done during a scale-down or StatefulSet deletion.
 
+The initial beta version did not set the controller flag in the owner
+reference. If a PVC has an owner reference to the expected StatefulSet or Pod
+object, and the controller flag is not set, it will be set, if there are no
+other controllers on the object. If there is another controller, the existing
+non-controller StatefulSet or Pod reference will be removed. That is, we will
+assume any such references are due to the older version of the controller.
+
 ### Version Skew Strategy
 There are only kube-controller-manager changes involved (in addition to the
 apiserver changes for dealing with the new StatefulSet field). Node components
```

Comment:
can we also update the implementation history?

Reply:
Done